packages/commands/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@dhis2/cypress-helpers",
+  "version": "1.0.0",
+  "description": "Cypress commands for dhis2 related apps",
+  "main": "build/cjs/lib.js",
+  "module": "build/es/lib.js",
+  "repository": "https://github.com/dhis2/cypress-helpers",
```

Comment:
This should point to this repository, `dhis2/cypress-helpers` doesn't exist.

You can use the last form of the `repository` field [documented here](https://docs.npmjs.com/files/package.json#repository) which allows you to specify a folder within the repo